turbo/adapter/ethapi/api.go

```diff
@@ -334,8 +334,13 @@ func RPCMarshalBlockExDeprecated(block *types.Block, inclTx bool, fullTx bool, b
 	}
 	fields["uncles"] = uncleHashes
 
-	if block.Withdrawals() != nil {
-		fields["withdrawals"] = block.Withdrawals()
+	if block.HeaderNoCopy().WithdrawalsHash != nil {
```

Comment:
A functionally equivalent way to write this line... i sort of prefer the `!= nil` version but they both work:
```suggestion
	if _, withdrawalsPresent := fields["withdrawalsRoot"]; withdrawalsPresent {
```